### ISSUE TITLE
Consider different approaches to scaffolding of non-visual experience

### ISSUE THREAD

**Issue by terracoda** [phetsims/quadrilateral#206]
Two sessions with blind science consultants using Quad reveals several of design issues with the current Voicing some related to scaffolding of ideas, some related to awkwardness Tab Navigation.

@terracoda to finish

**Comment by BLFiedler** [comment]
One thing @terracoda and I discussed yesterday was changing the tab order to land on side AB first. Manipulating a side while in a square gives you very easy access to rhombus, parallelogram, and rectangle which all have fairly simple descriptions.

Open question is if it goes 1) Side AB -> A -> B and then alternate or 2) Side AB -> B and continue alternating.

For 1), it might get a little weird to have it go side, vertex, vertex, side, vertex, side, vertex, side, vertex, vertex. For 2), it might be a bit weird to have A so late in the tab order.

Maybe an option is to have Side DA be first and then stick to the side/vertex alternating (corner A would be second)?

**Comment by terracoda** [comment]
We've implemented a Tab order that starts with Side DA and ends with Corner D.
We'll see how that works in next sessions.

We are also discussing other issues such as 
- the possibility of chunking in #244 , 
- the possibility of group navigation in #213.

In addition, we have implemented descriptions that describe 
- the width of the angles and the length of the sides  in #231 
Describing the angle slices and side units should give learners a benchmark for the qualitative comparisons.

I think we can close this issue and we will follow-up in the other issues mentioned above.
